ouroboros-network/ouroboros-network.cabal

```diff
@@ -343,9 +348,12 @@ test-suite test
                        -Wno-unticked-promoted-constructors
                        -fno-ignore-asserts
                        -threaded
+                       -rtsopts
+                       +RTS -T -RTS
```

Comment:
is the `-T` flag added to account for gc pauses, as mentioned [here](https://github.com/input-output-hk/ouroboros-network/pull/3463/files#diff-04e0d1becd2e1cfab7670b9121187b69a164d66820d45e8857d12ffdf180f59cR124)?